basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,226 @@
 const helpers = require('./project-1');
 
 // start testing!
+describe('project-1.js', () => {
+    
+    //TEST FOR FUNCTION ** multiplyByTen **
+    test('should multiply input number by 10', () => {
+        expect(helpers.multiplyByTen(10)).toBe(100);
+    })
+    
+    //TEST FOR FUNCTION ** subtractFive **
+    test('should subtract input by 5', () => {
+        const input = 10;
+        const expected = 5;
+        const result = helpers.subtractFive(input);
+
+        expect(result).toBe(expected);
+    })
+
+    //TEST FOR FUNCTION ** areSameLength **
+    describe('areSameLength', () => {
+        it('it should check length', () => {
+            const expected = true;
+            const actual = helpers.areSameLength('dog', 'abc');
+            expect(actual).toBe(expected);
+        })
+    })
+
+    //TEST FOR FUNCTION ** areEqual **
+    describe('areEqual', () => {
+        it('it should check numbers are equal?', () => {
+            const expected = true;
+            const actual = helpers.areEqual(25, 25);
+            expect(actual).toBe(expected);
+        })
+    })
+
+    //TEST FOR FUNCTION ** lessThanNinety **
+    describe('lessThanNinety', () => {
+        it('should check given number is lessThanNinety', () => {
+            const expected = true;
+            const actual = helpers.lessThanNinety(25);
+            expect(actual).toBe(expected);
+        })
+    })
+
+    //TEST FOR FUNCTION ** greaterThanFifty **
+    describe('greaterThanFifty', () => {
```

Comment:
You can test if the number is less than 50 as well and expect the value to be false.